### ISSUE TITLE
can not config with file in spring-boot

### ISSUE THREAD

**Issue by zhaozhi406** [redisson/redisson#737]
I want to configure redisson in a spring boot project, but encounter such exception:
```
Caused by: java.lang.Error: factory already defined
	at java.net.URL.setURLStreamHandlerFactory(URL.java:1112)
	at org.redisson.misc.URLBuilder.init(URLBuilder.java:43)
	at org.redisson.misc.URLBuilder.<clinit>(URLBuilder.java:35)
	at org.redisson.config.ConfigSupport.<init>(ConfigSupport.java:121)
	at org.redisson.config.Config.fromJSON(Config.java:515)
```
I searched java manual and found that's because setURLStreamHandlerFactory can only be called once in a given jvm. In my case, this method has been called by org.apache.catalina.webresources.TomcatURLStreamHandlerFactory() , so when I configure RedissonClient using following code, the exception came out:
```
@Bean(destroyMethod = "shutdown")
    public RedissonClient redissonClient() throws IOException {
        Resource resource = resourceLoader.getResource(config);
        Config cfg = Config.fromJSON(resource.getFile());
        return Redisson.create(cfg);
    }
```

Is there a way to avoid such exception?

**Comment by mrniko** [comment]
This bug already fixed in 2.7.2/3.2.2 versions https://github.com/redisson/redisson/issues/734

**Comment by zhaozhi406** [comment]
thank you, I changed to 3.2.2 and that's OK.
